js/model.js

```diff
@@ -48,14 +194,7 @@ class Square {
  * @returns {!GameBoard}
  */
 function createGameBoard(canvasHeight, canvasWidth) {
-  const gameBoard = {
-    startPointX: 0,
-    startPointY: 0,
-    gameHeight: 10,
-    gameWidth: 10,
-    dimensionsOfSquare: 0,
-    board: []
-  };
+  const gameBoard = new GameBoard(0, 0, 10, 10, 0, []);
```

Comment:
suggestion: break these out into module-level constants so it's easier to see what they refer to & to refactor.

Also a nit: having five number params in your constructor makes it easy to mix up the order and difficult to read.

One option is to make `GameBoard` take a params object, like

```
/** 
 * @param {{startPointX: number, startPointY: number, gameHeight: number, gameWidth: number, dimensionsOfSquare: number}} gameDimensions 
 * @param {!Array<!Array<!Square>>} board
*/
constructor({startPointX, startPointY, gameHeight, gameWidth, dimensionsOfSquare}, board) {
```

Reply:
Changed to be make use of the object and be more clean.